### PR TITLE
updating env var declaration

### PR DESCRIPTION
**Pull request by catalinaperalta**
These env vars will reference env vars defined in the integration test pipeline

### CODE REVIEW

File: eng/azure-pipelines.yml [hunk 1]
```diff
@@ -60,28 +60,3 @@ jobs:
...
-          go test -race -v -coverprofile=coverage.txt -covermode atomic ./... 2>&1 | go-junit-report > report.xml
-          gocov convert coverage.txt > coverage.json
-          gocov-xml < coverage.json > coverage.xml
-          gocov-html < coverage.json > coverage.html
-        workingDirectory: '$(sdkPath)'
-        displayName: 'Run Tests'
-      - script: |
-          gofmt -s -l -w . >&2
-        workingDirectory: '$(sdkPath)'
-        displayName: 'Format Check'
```
Comment (jhendrixMSFT) [inline eng/azure-pipelines.yml:74]: You should keep the format check.

Comment (catalinaperalta) [reply eng/azure-pipelines.yml:74]: Done

File: eng/integration-tests.yml [hunk 2]
```diff
@@ -31,12 +31,31 @@ steps:
...
       make destroy-sb
+    env:
+      ARM_SUBSCRIPTION_ID: $(go-eh-azure-subscription-id)
+      ARM_CLIENT_ID: $(go-eh-azure-client-id)
+      ARM_CLIENT_SECRET: $(go-eh-azure-client-secret)
+      ARM_TENANT_ID: $(go-eh-azure-tenant-id)
     displayName: 'Run Integration Tests'
+  - script: |
+      set -e
+      go test -race -v -coverprofile=coverage.txt -covermode atomic ./... 2>&1 | go-junit-report > report.xml
```
Comment (catalinaperalta) [inline eng/integration-tests.yml:46]: Make test-cover should include this, right?

Comment (jhendrixMSFT) [reply eng/integration-tests.yml:46]: Correct.  Be sure to destroy after.

Comment (catalinaperalta) [reply eng/integration-tests.yml]: ok good, removed and calling destroy correctly now.

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
Your subscription and tenant env vars are incorrect, remove the `-eh` part.

**Comment by catalinaperalta** [comment]
> Your subscription and tenant env vars are incorrect, remove the `-eh` part.

Thanks for catching that, was just looking into what happened, I've updated it.

**Pull request merged**